Mathlib/Topology/Algebra/Group/TopologicalAbelianization.lean

```diff
@@ -0,0 +1,63 @@
+/-
+Copyright (c) 2023 María Inés de Frutos-Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: María Inés de Frutos-Fernández
+-/
+import Mathlib.GroupTheory.Abelianization
+import Mathlib.Topology.Algebra.Group.Basic
+
+/-!
+# The topological abelianization of a group.
+
+This file defines the topological abelianization of a topological group.
+
+## Main definitions
+
+* `TopologicalAbelianization`: defines the topological abelianization of a group `G` as the quotient
+  of `G` by the topological closure of its commutator subgroup..
+
+## Main results
+- `instNormalCommutatorClosure` : the topological closure of the commutator of a topological group
+  `G` is a normal subgroup.
+
+## Tags
+group, topological abelianization
+
+-/
+
+variable (G : Type*) [Group G] [TopologicalSpace G] [TopologicalGroup G]
+
+instance instNormalCommutatorClosure : (commutator G).topologicalClosure.Normal :=
+  Subgroup.is_normal_topologicalClosure (commutator G)
+
+/-- The topological abelianization of `absoluteGaloisGroup`, that is, the quotient of
+  `absoluteGaloisGroup` by the topological closure of its commutator subgroup. -/
+def TopologicalAbelianization := G ⧸ Subgroup.topologicalClosure (commutator G)
```

Comment:
Maybe we can make it an `abbrev` so that just the CommGroup instance need to be added. We've just recently changed [AddCircle](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/Instances/AddCircle.html#AddCircle) to an abbrev, so this is a reasonable thing to do. I won't ask you to change Abelianization in this PR though :-)